src/evaluation_policy.jl

```diff
@@ -7,25 +7,28 @@ Interface for defining an evaluation policy
     returns the average reward of the current policy, the user can specify its own function 
     f to carry the evaluation, we provide a default basic_evaluation that is just a rollout. 
 """
-function evaluation(f::Function, policy::AbstractNNPolicy, env::AbstractEnvironment, n_eval::Int64, max_episode_length::Int64, verbose::Bool = false)
+function evaluation(f::Function, policy::AbstractNNPolicy, env::AbstractEnv, n_eval::Int64, max_episode_length::Int64, verbose::Bool = false)


```

Comment:
How did `^M` come in? Likely a `vim` issue?

Reply:
Yeah, I am not sure how the ^M is happening. I will correct it. Thanks